src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -1410,6 +1410,10 @@ private static Attribute NewCmdletBindingAttribute(AttributeAst ast)
                 {
                     result.RemotingCapability = s_attrArgToRemotingCapabilityConverter.Target(s_attrArgToRemotingCapabilityConverter, argValue);
                 }
+                else if (argumentName.Equals("HideCommonParameters", StringComparison.OrdinalIgnoreCase))
```

Comment:
Can we use `nameof` and `Ordinal`?

Reply:
No to Ordinal because users can type it in without proper casing: `[CmdletBinding(hideCommonParameters)]`.  
I'm not sure what you mean with `nameof` isn't that used to get the name of a specified type? I haven't created a new type so I don't see how I can use it.